### DISCUSSION TITLE
Renamed table but postgrest api endpoint remains?

### DISCUSSION THREAD

**Discussion by WesleyYue** [supabase/supabase#10286] [Questions] [answered]
I renamed a table, but the postgrest api endpoint still exists (except all fields are now optional). Is this a bug? How can I remove these endpoints?

**Comment by GaryAustin1** [comment, marked as answer]
Probably the schema cache
https://postgrest.org/en/stable/schema_cache.html#schema-cache
Running this in SQL editor should reset it.
`NOTIFY pgrst, 'reload schema'`

**Reply by WesleyYue** [reply]
Thanks for the quick response. Turns out it was actually a new view that was named the old table name and I didn't realize. Sorry about that!
